packages/hooks/src/useCreateNFT.ts

```diff
@@ -75,7 +80,7 @@ export enum CreateNftStep {
 }
 
 type createNftFn = (data: {
-  standard: Standard
+  collection: string
```

Comment:
Would be good to keep the naming consistent with API:
```suggestion
  collectionAddress: string
```